app/models/mentor_questionnaire.rb

```diff
@@ -7,6 +7,8 @@ class MentorQuestionnaire < ApplicationRecord
   validates :year_started_ruby, presence: true
   validates :has_mentored_before, presence: true
   validates :mentoring_reason, presence: true
-  validates :preferred_style_career, presence: true
-  validates :preferred_style_code, presence: true
+  validates :preferred_style_career, presence: true, inclusion: {in: [true, false]}
+  validates :preferred_style_code, presence: true, inclusion: {in: [true, false]}
+
```

Comment:
Added inclusion as local tests wasn't letting me save this as `false`

Reply:
oh wow! TIL presence is truthy. 

Removed validations and updated the test. Thanks